### PR TITLE
Fix issues with building the project.

### PR DESCRIPTION
**Pull request by jbellmann**
- add initial '.travis.yml' for continuous-integration on Travis-CI
- remove parts not needed for now on 'build.gradle'
- utilize [Gradle-Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) to use same version of gradle on every build
- update README.md on how to use Gradle-Wrapper
- update '.gitignore' with output-folders created by the Gradle build

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.2.RELEASE") {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.5.RELEASE") {
             exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
         }
     }
```
Comment (MALPI) [inline build.gradle]: I just removed that, since it's not needed.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Thu Jun 30 15:08:56 CEST 2016
```
Comment (MALPI) [inline gradle/wrapper/gradle-wrapper.properties:1]: gradle wrapper shouldn't be committet

File: .gitignore [hunk 3]
```diff
@@ -54,3 +54,7 @@ fabric.properties
 *.jar
 *.war
 *.ear
+
+# Gradle build-folder
+build
+.gradle
```
Comment (MALPI) [inline .gitignore:60]: add gradle/*\* here to avoid comitting the gradle wrapper

**Comment by MALPI** [comment]
Sorry for conflicting...

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (jbellmann) [inline gradle/wrapper/gradle-wrapper.properties:1]: Can you explain to me please why you think Gradle-Wrapper is a bad idea?

Comment (MALPI) [inline gradle/wrapper/gradle-wrapper.properties:1]: Actually I just read how the travis works, and as far as I got it, it builds with the wrapper if it's comitted. So i think it's fine.

File: .gitignore (hunk 3 above)
Comment (jbellmann) [inline .gitignore:60]: According to your last comment above this can stay as is, right?

Comment (MALPI) [inline .gitignore:60]: true

**Pull request merged**